var/spack/repos/builtin/packages/cryptopp/package.py

```diff
@@ -28,4 +28,4 @@ def url_for_version(self, version):
         version_tuple = tuple(v for v in iter(version))
         version_string = reduce(lambda vs, nv: vs + str(nv), version_tuple, "")
```

Comment:
Sure, that's no problem at all!  I made this adjustment in my most recent commit.